app/controllers/ErrorController.scala

```diff
@@ -0,0 +1,14 @@
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{AbstractController, Action, AnyContent, ControllerComponents, Request}
+
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class ErrorController @Inject()(val cc: ControllerComponents)(implicit val ec: ExecutionContext)  extends AbstractController(cc) with I18nSupport {
+  def error(message: String): Action[AnyContent] = Action { implicit request: Request[AnyContent] =>
+    Ok(views.html.error(""))
```

Comment:
Should the `Ok` be a server error, or handle multiple types of errors? It's a bit confusing to return a 200 OK if there's an error, and it can lead to things like error pages being cached.